packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -93,83 +96,110 @@ export const styles = css`
     flex-direction: column;
     justify-content: center;
     text-align: center;
-    white-space: normal;
     height: 44px;
     width: 44px;
     border-radius: ${borderRadiusMedium};
+    box-sizing: border-box;
+  }
+  :host .secondary-panel-today {
+    color: ${colorNeutralForeground2Selected};
+    background-color: ${colorBrandBackgroundInvertedSelected};
+  }
+  :host .secondary-panel-cell-outer:not(.secondary-panel-today):not(.secondary-panel-selected):hover {
+    background-color: ${colorBrandBackgroundInvertedHover};
+    color: ${colorNeutralForeground1Static};
+  }
+  :host .secondary-panel-selected {
+    color: ${colorNeutralForeground1Static};
+    background-color: ${colorBrandBackgroundInvertedSelected};
+  }
+  :host .secondary-panel-cell-outer:not(.secondary-panel-today):not(.secondary-panel-selected):active {
+    background-color: ${colorBrandBackgroundInvertedSelected};
+    color: ${colorNeutralForeground1Static};
+  }
+  :host .day {
+    z-index: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    line-height: 32px;
+    width: 32px;
+    position: relative;
+    border-radius: ${borderRadiusMedium};
+  }
+  :host .interact .today:not(.inactive) {
+    color: ${colorNeutralForegroundStaticInverted};
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    z-index: 0;
+  }
+  :host .interact .today:not(.inactive)::after {
```

Comment:
Does this work well in theme: Web dark? How does it look when you set your OS to using a high-contrast mode?

Reply:
The hover background colors don't appear on high contrast but I'm using CSS tokens. Would this be due to the CSS token not supporting hc?